speech/cloud-client/src/main/java/com/example/speech/Recognize.java

```diff
@@ -55,6 +57,11 @@
 
 public class Recognize {
 
+  // Creating shared object
+  private static volatile BlockingQueue<byte[]> sharedQueue = new LinkedBlockingQueue();
+  private static TargetDataLine targetDataLine;
+  private static int BYTES_PER_BUFFER = 6400; // buffer size in bytes
```

Comment:
Is it possible to move these to inside the function?

That way they are included as part of the region tags include statement. If not, we may need to create a separate class file, but no need to do arg parsing. You could follow the proposed new style here: https://github.com/GoogleCloudPlatform/java-docs-samples/pull/1232

Reply:
Switched to the new format in the latest commit. Also needed to move it to it's own class file since that looks like the best approach. Please let me know if you would like me to further look into keeping it in the Recognize file itself.